test/sharness/t0110-gateway.sh

```diff
@@ -139,11 +139,32 @@ test_expect_success "refs IPFS directory file through readonly API succeeds" '
   test_curl_gateway_api "refs?arg=$HASH2/test"
 '
 
-test_expect_success "test gateway api is sanitized" '
-  for cmd in "add" "block/put" "bootstrap" "config" "dht" "diag" "dns" "get" "id" "mount" "name/publish" "object/put" "object/new" "object/patch" "pin" "ping" "refs/local" "repo" "resolve" "stats" "swarm"  "file" "update" "version" "bitswap"; do
+for cmd in add  \
```

Comment:
I removed `get`, `version`, `resolve`, and `dns` as these are now exposed on the gateway.